File: Python/Fibonacci_Number.py
print("Fibonacci Number Checker\n")
n = int(input("Enter the number : "))
flag = False
a=1
b=1
c=0
while n>=b:
	if b==n:
		flag=True
		break
	c=b
	b=a+b
	a=c
if flag:
	print(f"{n} is a fibonacci number.")
else:
	print(f"{n} is not a fibonacci number.")

File: Java/src/Fibonacci_Number.java
/**
 * Fibonacci Number Checker
 * 
 * @author Jaysmito Mukherjee 
 * @version 2019-A-012
 */
import java.util.*;
public class Fibonacci_Number
{
    public static void main()
    {
        System.out.print("\f");
        Scanner sc=new Scanner(System.in);
        System.out.print("Fibonacci Number Checker\n~~~~~~~~~ ~~~~~~ ~~~~~~~\nEnter the number: ");
        int n=sc.nextInt();
        int a=-1,b=1,c=0;
        boolean flag=false;
        while(c<=n)
        {
            c=a+b;
            if(c==n)
            {
                flag=true;
                break;
            }
            a=b;
            b=c;
        }
        if(flag)
            System.out.print(n+" is a Fibonacci number.");
        else
            System.out.print(n+" is not a Fibonacci number.");        
    }
}

